go.mod

```diff
@@ -6,38 +6,38 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/brancz/kube-rbac-proxy v0.8.0
 	github.com/docker/distribution v2.7.1+incompatible
-	github.com/evanphx/json-patch/v5 v5.2.0
+	github.com/evanphx/json-patch/v5 v5.5.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-kit/kit v0.10.0
 	github.com/go-openapi/swag v0.19.15
 	github.com/gogo/protobuf v1.3.2
-	github.com/google/go-cmp v0.5.5
+	github.com/google/go-cmp v0.5.6
 	github.com/hashicorp/go-version v1.3.0
 	github.com/kylelemons/godebug v1.1.0
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/oklog/run v1.1.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus-community/prom-label-proxy v0.2.0
+	github.com/prometheus-community/prom-label-proxy v0.3.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.48.1
 	github.com/prometheus-operator/prometheus-operator/pkg/client v0.48.1
-	github.com/prometheus/alertmanager v0.21.1-0.20210422101724-8176f78a70e1
-	github.com/prometheus/client_golang v1.10.0
-	github.com/prometheus/common v0.23.0
-	github.com/prometheus/prometheus v1.8.2-0.20210518124745-db7f0bcec27b
+	github.com/prometheus/alertmanager v0.22.2
+	github.com/prometheus/client_golang v1.11.0
+	github.com/prometheus/common v0.29.0
+	github.com/prometheus/prometheus v1.8.2-0.20210519120135-d95b0972505f
```

Comment:
can we make it the same commit ID as the latest supported version of Prometheus?